backend/src/routes/__tests__/index.test.js

```diff
@@ -0,0 +1,21 @@
+import serviceRouter from '../service';
+import comboRouter from '../combo';
+
+const useSpy = jest.fn();
```

Comment:
### Problem
Variable name may give a confusion: I think your intension of choosing variable name like this is that mock function let us to spy on the behavior of a function. However, it may give the confusion of with actual spyOn.

### Fix
Change the variable name.

### Comment
It is really minor concern, you don't have to do. I just suggest.